requirements.txt

```diff
@@ -1,3 +1,3 @@
 tornado>=6.0
-aiocoap
+git+https://github.com/chrysn/aiocoap.git@0.4b3
```

Comment:
Why do you need this specific version ?

Reply:
There is a new version now that was release, I can use that one. I'm not sure but It doesn't not with current RIOT master without.

Reply:
Still figuring out things with the new aiocoap version, I'll PR once that is resolved.